tensorflow/core/kernels/data/experimental/snapshot_util.h

```diff
@@ -103,11 +103,11 @@ class Reader {
   // The reader input buffer size is deliberately large because the input reader
   // will throw an error if the compressed block length cannot fit in the input
   // buffer.
-  static constexpr const int64 kSnappyReaderInputBufferSizeBytes =
-      1 << 30;  // 1 GiB
+  //static constexpr const int64 kSnappyReaderInputBufferSizeBytes =
```

Comment:
C++14 requires this to be defined somewhere. C++17 does not, but we still need to support C+14.

Reply:
I moved the definition to the corresponding .cc file to avoid an import error.